app/nasa_image/page.jsx

```diff
@@ -0,0 +1,23 @@
+import Link from "next/link";
+import styles from "./page.module.css";
+
+export default function EpicImageHome() {
+  return (
+    <div className={styles.container}>
+      <h1 className={styles.title}>No date provided</h1>
+      <p>Please select a date to view the NASA image.</p>
+      <p className={styles.info}>
+        For example, to view the image for October 31, 2015, navigate to:
+        <br />
+        <code>http://localhost:3000/nasa_image/2015-10-31</code>{" "}
+      </p>
+
+      <p className={styles.info}>
+        You can also click the link below to view the NASA image for that date:
```

Comment:
Super nice! It would be super interesting to see if we could have a date picker here so that the date isn't hardcoded though.